docs/packages/pkg/soil.rst

```diff
@@ -0,0 +1,21 @@
+.. spelling::
+
+    soil
+
+.. index:: 
+  single: graphics ; soil
+
+.. _pkg.soil:
+
+soil
+====
+
+-  https://www.lonesock.net/soil.html
+-  `Official <https://github.com/frequem/Simple-OpenGL-Image-Library>`__
```

Comment:
Doesn't look like an official repository

Reply:
Official repository doesn't exist. SOIL library was copied and posted to github by @kbranigan from the http://lonesock.net/soil.htm. So 'official' there is only the web page.

Reply:
I changed it to be a bit more clear and added kbranigan's version as 'GitHub' and mine as 'Hunterized'